react/week2/todo-list-app/src/components/TodoListItem.js

```diff
@@ -0,0 +1,112 @@
+import React, { Component } from "react";
+
+export default class TodoList extends Component {
+  state = {
+    todoList: [
+      {
+        id: 1,
+        description: "Get out of bed",
+        checked: false,
+      },
+      {
+        id: 2,
+        description: "Brush teeth",
+        checked: false,
+      },
+      {
+        id: 3,
+        description: "Eat breakfast",
+        checked: false,
+      },
+    ],
+    todoAddNewItem: " ",
+  };
+
+  addTodo = (todoAddNewItem) => {
+    this.setState({
+      todoList: [
+        ...this.state.todoList,
+        {
+          id: 4,
+          name: todoAddNewItem,
+          description: "random text",
+          completed: false,
+        },
+      ],
+    });
+  };
+
+  deleteTodo = (index) => {
```

Comment:
If i made a function that deleted todoitems i would make sure its easy to use the function. Index could mean lost of things, but i guess you mean id? Or do you mean the index it is in an array or in the rendered list?

```suggestion
  deleteTodo = (itemId) => {
```